packages/calypso-e2e/src/lib/pages/media-page.ts

```diff
@@ -173,4 +179,15 @@ export class MediaPage {
 			);
 		}
 	}
+
+	/* Modal mode */
+
+	/**
+	 *
```

Comment:
Looks like this jsdoc header is missing as an FYI!

Reply:
Ah my bad, thanks!